src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/form.html.twig

```diff
@@ -646,23 +660,26 @@
         {% if data.resolved_options is defined %}
         <h3>
             <a class="toggle-button" data-toggle-target-id="{{ data.id }}-resolved_options" href="#">
-                Resolved Options
-                <span class="toggle-icon"></span>
+                Resolved Options <span class="toggle-icon"></span>
             </a>
         </h3>
 
         <div id="{{ data.id }}-resolved_options" class="hidden">
-            <table>
-                <tr>
-                    <th width="180">Option</th>
-                    <th>Value</th>
-                </tr>
+            <table class="auto-layout">
+                <thead>
+                    <tr>
+                        <th width="180">Option</th>
+                        <th>Value</th>
+                    </tr>
+                </thead>
+                <tbody>
                 {% for option, value in data.resolved_options %}
                 <tr>
                     <th>{{ option }}</th>
-                    <td><pre>{{ value }}</pre></td>
```

Comment:
Yes, because by default we treat all table cells as `<pre>` elements (but breaking long lines to prevent issues).